api/hub/src/datasources/moderation-decision.ts

```diff
@@ -250,14 +261,17 @@ class ModerationDecisionAPI extends DataSource {
     if (await queryCache.has(decisionCache)) {
       return queryCache.get(decisionCache);
     }
-    const decision = await this.getDecision(contentID);
+    let finalDecision = await this.getDecision(contentID);
```

Comment:
this can be `const` if you're not reasigning `finalDecision`

Reply:
I'm calling `Object.assign()` later to add extra fields, so I can't make it a const.